cmd/ingester/app/consumer/rate_limiter_test.go

```diff
@@ -57,12 +57,12 @@ func TestRateLimiterNoLimit(t *testing.T) {
 	r := newRateLimiter(creditsPerSecond)
 	defer r.Stop()
 
-	const maxInterval = time.Microsecond
+	const maxInterval = time.Millisecond
 	acquireStart := time.Now()
 	for i := 0; i < iterations; i++ {
```

Comment:
I don't see any references to `r` in this loop - what is it testing?

Reply:
Good catch. Guess I missed the most important line here.

Reply:
Fixed.